[0m16:04:55.822820 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ac4b640>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10bddbc10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10bddbd00>]}


============================== 16:04:55.829373 | c98a37e1-438e-40c7-95b0-a3ce9b8caa7d ==============================
[0m16:04:55.829373 [info ] [MainThread]: Running with dbt=1.5.2
[0m16:04:55.830112 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/Users/albertstanley/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m16:04:55.831541 [debug] [MainThread]: Starter project path: /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dbt/include/starter_project
[0m16:05:44.691252 [error] [MainThread]: Encountered an error:

[0m16:05:44.696591 [error] [MainThread]: Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dbt/cli/requires.py", line 86, in wrapper
    result, success = func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dbt/cli/requires.py", line 71, in wrapper
    return func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dbt/cli/main.py", line 457, in init
    results = task.run()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dbt/task/init.py", line 307, in run
    self.create_profile_from_target(adapter, profile_name=project_name)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dbt/task/init.py", line 180, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dbt/task/init.py", line 164, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/dbt/task/init.py", line 130, in generate_target_from_input
    target[key] = click.prompt(
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/click/termui.py", line 166, in prompt
    value = prompt_func(prompt)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/click/termui.py", line 149, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m16:05:44.698608 [debug] [MainThread]: Command `dbt init` failed at 16:05:44.697635 after 48.93 seconds
[0m16:05:44.699188 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ac4b640>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1252570a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x125256c80>]}
[0m16:05:44.699729 [debug] [MainThread]: Flushing usage events
[0m16:06:05.048382 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f433610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1105cbd00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1105cbdf0>]}


============================== 16:06:05.055679 | 705c4294-f047-42b8-badf-7aa65ed14eb6 ==============================
[0m16:06:05.055679 [info ] [MainThread]: Running with dbt=1.5.2
[0m16:06:05.056416 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/albertstanley/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m16:06:05.057717 [info ] [MainThread]: A project called jaffle_shop already exists here.
[0m16:06:05.058646 [debug] [MainThread]: Command `dbt init` succeeded at 16:06:05.058468 after 0.05 seconds
[0m16:06:05.059111 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f433610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1105cbe20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1105cba30>]}
[0m16:06:05.059563 [debug] [MainThread]: Flushing usage events
